var/spack/repos/builtin/packages/steps/package.py

```diff
@@ -35,10 +37,19 @@ class Steps(CMakePackage):
     depends_on("py-unittest2", type=("build", "test"))
     depends_on("python")
 
+    depends_on("easyloggingpp", when="-bundle")
+    depends_on("random123", when="-bundle")
+    depends_on("sundials@:2.99.99+int64", when="-bundle")
```

Comment:
could you use `~` instead of `-` in when clause? sometime it's confusing because variant name itself can have `-`.

Reply:
sure